StringCalculator.js

```diff
@@ -0,0 +1,44 @@
+const StringCalculator = (args) => {
+    const inputs = args.split(" ");
+
+    const calculator = (number, input) => {
+        if (number === 0) return Number.parseInt(input[number]);
+
+        return branchMethod(number, input);
+    };
+
+    const branchMethod = (number, input) => {
+        switch (input[number]) {
```

Comment:
```suggestion
        const operand = input[number];
        switch (operand) {
```
이런 식으로 해준다면 `input[number]`가 의미하는바가 무엇인지 한 번에 알 수 있을 것 같은데 이렇게 하는 방법이 무의미한 메모리공간 낭비라고 생각하시나요 아니면 가독성 좋은 방법이라고 생각하시나요? ㅎㅎㅎㅎ